uktrade/great-international-ui
# Display image or video on international article page

This PR displays either a video or image (with video taking precedence) on wagtail pages using the template `InternationalArticlePage`.

It also updates the video transcript styling to be inline with a recent accessibility audit. The transcript is no longer scrollable and follows standard gov uk styling for the summary component.

**To test**
- Pull down this branch from directory cms - `feature/article-page-video`
- Ensure you have an article video and/or image uploaded at http://cms.trade.great:8010/admin/pages/829/edit/
- You need a video with a transcript - try 'Test intro video'
- Go to http://international.trade.great:8012/international/content/investment/how-we-can-help/freeports-in-the-uk/
- You should see the video with transcript or image 

**Before**

Video transcript
![Screenshot 2023-07-04 at 15 57 35](https://github.com/uktrade/great-international-ui/assets/22460823/a23fdbf0-395d-4bfd-9336-0a4ac860ffb0)

Article page
![Screenshot 2023-07-04 at 15 58 14](https://github.com/uktrade/great-international-ui/assets/22460823/956ec5b4-736d-4694-b107-57e3ce208507)

**After**

Transcript open
![Screenshot 2023-07-04 at 15 55 51](https://github.com/uktrade/great-international-ui/assets/22460823/f76b7352-7e00-4d37-8793-cbc07fd42495)


Transcript hidden
![Screenshot 2023-07-04 at 15 55 57](https://github.com/uktrade/great-international-ui/assets/22460823/82abec62-9f1f-4943-8ec2-7af6cb41a2d4)


### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-811

### Reviewing help


### Housekeeping


### Merging
